test/backend.test.js

```diff
@@ -236,7 +239,9 @@ describe('Full walkthrough', function () {
     it('Should create mailboxes for Alice and Bob and subscribe', async function () {
       // Object.values(users).forEach(async user => await createMailboxFor(user))
       await createMailboxFor(users.alice)
+      console.log('alice ok')
       await createMailboxFor(users.bob)
+      console.log('bob ok')
```

Comment:
I don't think these console.logs should be here?

Reply:
Good catch! :sweat_smile: